lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivation/RandomSpec.hs

```diff
@@ -72,17 +95,42 @@ prop_keyDerivation seed encPwd ix1 ix2 =
     accXPrv = deriveAccountPrivateKey encPwd rootXPrv ix1
     addrXPrv = deriveAddressPrivateKey encPwd accXPrv ix2
 
+prop_derivationPathRoundTrip
```

Comment:
don't we want to separate this property into two : one checking roundtrip other that another root public keys cannot serve to decode the same encoded bytestring?

Reply:
This is the way I was shown in quviq training. And the generator/test case is quite slow, so I don't want to make it twice as slow to test.